cmd/packaging.go

```diff
@@ -0,0 +1,472 @@
+		"summary: " + assertInFlutterProject().Description,
+		"description: |",
+		"  " + assertInFlutterProject().Description,
+		"confinement: devmode",
+		"grade: devel",
+		"apps:",
+		"  " + escapedProjectName(projectName) + ":",
+		"    command: " + projectName,
+		"    desktop: local/" + projectName + ".desktop",
+		"parts:",
+		"  desktop:",
+		"    plugin: dump",
+		"    source: snap",
+		"  assets:",
+		"    plugin: dump",
+		"    source: assets",
+		"  app:",
+		"    plugin: dump",
+		"    source: build",
+		"    stage-packages:",
+	}
+	for _, dependency := range linuxPackagingDependencies {
+		snapcraftFileContent = append(snapcraftFileContent, "      - "+dependency)
+	}
+
+	for _, line := range snapcraftFileContent {
+		if _, err := snapcraftFile.WriteString(line + "\n"); err != nil {
+			fmt.Printf("hover: Could not write snapcraft.yaml: %v\n", err)
+			os.Exit(1)
+		}
+	}
+	err = snapcraftFile.Close()
+	if err != nil {
+		fmt.Printf("hover: Could not close snapcraft.yaml: %v\n", err)
+		os.Exit(1)
+	}
+
+	desktopFilePath, err := filepath.Abs(filepath.Join(snapLocalDirectoryPath, projectName+".desktop"))
+	if err != nil {
+		fmt.Printf("hover: Failed to resolve absolute path for desktop file %s: %v\n", desktopFilePath, err)
+		os.Exit(1)
+	}
+	desktopFile, err := os.Create(desktopFilePath)
+	if err != nil {
+		fmt.Printf("hover: Failed to create desktop file %s: %v\n", desktopFilePath, err)
+		os.Exit(1)
+	}
+	desktopFileContent := []string{
+		"[Desktop Entry]",
+		"Encoding=UTF-8",
+		"Version=" + assertInFlutterProject().Version,
+		"Type=Application",
+		"Terminal=false",
+		"Exec=/" + projectName,
+		"Name=" + projectName,
+		"Icon=/icon.png",
+	}
+
+	for _, line := range desktopFileContent {
+		if _, err := desktopFile.WriteString(line + "\n"); err != nil {
+			fmt.Printf("hover: Could not write %s.desktop: %v\n", projectName, err)
+			os.Exit(1)
+		}
+	}
+	err = desktopFile.Close()
+	if err != nil {
+		fmt.Printf("hover: Could not close %s.desktop: %v", projectName, err)
+		os.Exit(1)
+	}
+
+	gitignoreFilePath, err := filepath.Abs(filepath.Join(snapDirectoryPath, ".gitignore"))
+	if err != nil {
+		fmt.Printf("hover: Failed to resolve absolute path for .gitignore file %s: %v\n", gitignoreFilePath, err)
+		os.Exit(1)
+	}
+	gitignoreFile, err := os.Create(gitignoreFilePath)
+	if err != nil {
+		fmt.Printf("hover: Failed to create .gitignore file %s: %v\n", gitignoreFilePath, err)
+		os.Exit(1)
+	}
+	gitignoreFileContent := []string{
+		"assets",
+		"build",
+		"*.snap",
+	}
+
+	for _, line := range gitignoreFileContent {
+		if _, err := gitignoreFile.WriteString(line + "\n"); err != nil {
+			fmt.Printf("hover: Could not write .gitignore: %v\n", err)
+			os.Exit(1)
+		}
+	}
+	err = gitignoreFile.Close()
+	if err != nil {
+		fmt.Printf("hover: Could not close .gitignore: %v", err)
+		os.Exit(1)
+	}
+
+	fmt.Printf("hover: You now can package the snap using `hover build %s`\n", packagingFormat)
```

Comment:
Notify the user of the creation of `go/packaging/linux-snap` (that way, he can add it to git)